services/webtheme/webtheme.go

```diff
@@ -107,19 +108,20 @@ func parseThemeMetaInfo(fileName, cssContent string) *ThemeMetaInfo {
 
 func initThemes() {
 	availableThemes = nil
+	defaultTheme := setting.Config().Theme.DefaultTheme.Value(context.Background())
```

Comment:
`initThemes` is called in "once", but `defaultTheme` can be dynamically changed now, so how could the logic be right for a changed "default theme"?